UX-TODO.md

```
@@ -0,0 +1,58 @@
+# Hydra UI UX / Design Refactor Backlog
+
+Status Legend: [ ] Pending  [~] In Progress  [x] Complete  [!] Blocked
+
+## P0 Foundational
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)